woocommerce/wc-admin
# Migrated "import" store to "wp.data"

Fixes #3382

This PR migrates the `import` store to `wp.data`.

Also added error handling.

### Screenshots
![screenshot-one wordpress test-2020 08 13-11_40_08](https://user-images.githubusercontent.com/1314156/90148599-deeb1e80-dd59-11ea-885e-f3a968e89768.png)

### Detailed test instructions:
- Go to `WooCommerce` | `Analytics` | `Settings` (URL: `/wp-admin/admin.php?page=wc-admin&path=/analytics/settings`)
- If you need to add orders to test you should turn `WC-Admin` off, add a new order and turn it on again.
- The count of `customers` and `orders and refunds` available to import should be visible.
- Verify no HTTP/JS errors occur.
- Press `Start`.
- Everything should work normally.

Re-import data
- After the process finishes, it should show the button `Re-import data`. Press it.
- Start the process again.
- Refresh the browser.
- The button `Stop` should be visible until the importation finishes.

_Be careful when you press `Delete Previously Imported Data`. It could show a false positive since it's a process that runs under the hood_



### Changelog Note:
Dev: migrate import store to wp.data (don't include this in plugin changelog)
